osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -135,97 +118,96 @@ void removeOldInfo()
                     return;
                 }
 
-                LoadComponentAsync(loadingInfo = new BufferedWedgeInfo(beatmap, ruleset.Value, mods.Value, beatmapDifficulty.Value ?? new StarDifficulty())
+                LoadComponentAsync(loadingInfo = new BeatmapInfoWedgeContainer(beatmap, ruleset.Value)
                 {
                     Shear = -Shear,
-                    Depth = Info?.Depth + 1 ?? 0
+                    Depth = Container?.Depth + 1 ?? 0,
                 }, loaded =>
                 {
                     // ensure we are the most recent loaded wedge.
                     if (loaded != loadingInfo) return;
 
                     removeOldInfo();
-                    Add(Info = loaded);
+                    Add(Container = loaded);
                 });
             }
         }
 
-        protected override void Dispose(bool isDisposing)
+        public class BeatmapInfoWedgeContainer : Container
```

Comment:
The ordering changes in this file are making this diff unnecessarily large and making it harder to review the pull (have to cross-check implementations for changes). Would rather see member order restored to the original where it's been altered to decrease diff size.